config/manifests/vllm/sim-deployment.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
       - name: vllm-sim
-        image: ghcr.io/llm-d/llm-d-inference-sim:v0.1.2
+        image: ghcr.io/llm-d/llm-d-inference-sim:latest
```

Comment:
this one is a bit more problematic. there is another open PR that was trying to do the same.
see my comment there if your'e interesting in more details:
https://github.com/kubernetes-sigs/gateway-api-inference-extension/pull/1140

I think we can keep this PR focused on the cpu example update to get it merged quickly.

Reply:
yep updated